### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by nithishkumarradhakrishnan** [community/community#38340] [New to GitHub]
What type the GitHub is used to us

**Comment by ossanna16** [comment]
Hi there @nithishkumarradhakrishnan and welcome to our community!

You can read [this post](https://github.com/orgs/community/discussions/30514) to help you learn more about GitHub. Please let me know if it's helpful!

**Comment by queenofcorgis** [comment]
Hey there @nithishkumarradhakrishnan , we're happy you're here! You are more likely to get a useful response if you are explicit about what your questions entails, giving a few more details might help someone give you a nudge in the right direction.
